docs/Modules/index.mdx

```diff
@@ -67,7 +67,7 @@ Following is the list of Validation Modules:
 
 - [**ECDSA Ownership Module**](ecdsa.mdx): This module is widely adopted as a validation module for Biconomy smart accounts. It can seamlessly integrate with MPC providers such as Web3Auth, abstracting EOA Private Key storage and enabling a web2-like experience such as email login.
 - [**Multichain ECDSA Validator Module**](multichain.mdx): This module significantly improves UX for deploying and setting up Smart Accounts on several chains. It is an extension of the ECDSA Module enabling to dispatch multiple userOps on different chains using a single signature.
-- [**Session key Manager Validation Module**](sessionvalidationmodule.mdx): This module enables the use of sessions to execute transactions. It verifies whether a given user operation adheres to the permissions defined within the session key and confirms that the operation has been signed by that session key. This can only be used as an active validation module.
+- [**Session key Manager Validation Module**](./sessions/sessionvalidationmodule.mdx): This module enables the use of sessions to execute transactions. It verifies whether a given user operation adheres to the permissions defined within the session key and confirms that the operation has been signed by that session key. This can only be used as an active validation module.
```

Comment:
lets just call it Session Key Manager module. 
could be confusing with SKM

Reply:
> It verifies whether a given user operation adheres to the permissions defined within the session key and confirms that the operation has been signed by that session key

It verifies that session is on-chain and the operation has been signed by that session key and whether a given user operation adheres to the permissions defined within the session key using sub-modules contracts called session validation modules.